lib/gsheets/gsheets.js

```diff
@@ -1,21 +1,25 @@
-var fs = require('fs');
-var readline = require('readline');
-var google = require('googleapis');
-var GoogleAuth = require('google-auth-library');
-var process = require('process');
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
as a class works for me. this was originally some copypaste code from their examples.. but yeah i'm a fan of cleaning it up. :)